aikau/src/main/resources/alfresco/renderers/css/Thumbnail.css

```diff
@@ -1,48 +1,52 @@
 .alfresco-renderers-Thumbnail {
-   width: 100px;
+   width: @thumbnail-width;
+   height: @thumbnail-height;
    position: relative; /* This is required for absolutely positioned overlays */
-}
 
-.alfresco-renderers-Thumbnail span.link
-{
-   background: url(./images/link-32.png) no-repeat;
-   height: 32px;
-   width: 32px;
-   position: absolute;
-}
+   .alfresco-renderers-Thumbnail__frame {
+      display: inline-block;
+      height: @thumbnail-height;
+      width: @thumbnail-width;
+      line-height: @thumbnail-height;
+      text-align: center;
+   }
 
-.alfresco-renderers-Thumbnail span.link.hidden
-{
-   display: none;
-}
+   .alfresco-renderers-Thumbnail__image {
+      cursor: pointer;
+      max-height: ~'calc(100% - @thumbnail-margin - @thumbnail-margin - @thumbnail-border-width)';
```

Comment:
I can't remember why we allow `calc` when it doesn't work in IE8? I thought it was because it didn't matter when we'd previously used it, but this would presumably look bad in IE8?? There may be another reason entirely though.